### PR TITLE
Add `spin` animation. Rotate on the three axis.

### PR DESCRIPTION
**Pull request by phimage**
Use an enum for Core Animation key path.

### CODE REVIEW

**Comment by phimage** [comment]
spin

File: CHANGELOG.md [hunk 1]
```diff
@@ -13,16 +13,17 @@ N/A
 
 - Add new mask types `.drop`, `.plusSign`, `.moon`. [#519](https://github.com/IBAnimatable/IBAnimatable/pull/519) by [@phimage](https://github.com/phimage)
 - Add new mask types `.heart`, `.gear`, `.ring`, `.superEllipse`. [#518](https://github.com/IBAnimatable/IBAnimatable/pull/518) by [@phimage](https://github.com/phimage)
-- Add suppport for radial gradient. Currently not working with `startPoint`. [#527](https://github.com/IBAnimatable/IBAnimatable/pull/527) by [@tbaranes](https://github.com/tbaranes)
+- Add support for radial gradient. Currently not working with `startPoint`. [#527](https://github.com/IBAnimatable/IBAnimatable/pull/527) by [@tbaranes](https://github.com/tbaranes)
+- Animate TabBarItem image view when clicking on it. [#539](https://github.com/IBAnimatable/IBAnimatable/pull/539) by [@phimage](https://github.com/phimage)
+- Add `compound` animation type to do `sequential` or `parallel` animations. [#520](https://github.com/IBAnimatable/IBAnimatable/pull/520) by [@phimage](https://github.com/phimage)
```
Comment (JakeLin) [inline CHANGELOG.md:18]: 👍  I didn't notice they were on the wrong spots.

File: Sources/Protocols/Animatable/Animatable.swift [hunk 2]
```diff
@@ -217,7 +219,7 @@ fileprivate extension UIView {
               configuration: AnimationConfiguration,
               completion: AnimatableCompletion? = nil) {
     CALayer.animate({
-      let animation = CABasicAnimation(keyPath: "transform.rotation")
+      let animation = CABasicAnimation(keyPath: .rotation)
```
Comment (JakeLin) [inline Sources/Protocols/Animatable/Animatable.swift:222]: Great idea 👍

**Review by JakeLin — APPROVED** [review]
👍 very cool feature, thanks 🎉

**Pull request merged**